### PR TITLE
9093 refactoring styles approach

### PR DESCRIPTION
**Pull request by codyseibert**
blocked until batch 17
depends on https://github.com/flexion/ef-cms/pull/9444

the styling for the PDFs was messy.  This change is to split up the styling into multiple scss files to make this more managable.

### CODE REVIEW

File: shared/createModule.js [hunk 1]
```diff
@@ -1,6 +1,13 @@
 /* eslint-disable spellcheck/spell-checker */
 const fs = require('fs');
 
+const sass = require('sass');
+
+const { css } = sass.compile(
+  'shared/src/business/utilities/htmlGenerator/index-main.scss',
+);
+fs.writeFileSync('shared/src/business/utilities/htmlGenerator/index.scss', css);
+
```
Comment (TomElliottFlexion) [inline shared/createModule.js:10]: Pretty cool stuff

File: web-client/src/app.jsx [hunk 2]
```diff
@@ -115,6 +115,7 @@ import { wasAppLoadedFromACognitoLogin } from './utilities/wasAppLoadedFromACogn
 import { wasLoginUsingTokenInUrl } from './utilities/wasLoginUsingTokenInUrl';
 import { withAppContextDecorator } from './withAppContext';
 
+import { NoticeOfTrialIssued } from '../../shared/src/business/utilities/pdfGenerator/documentTemplates/NoticeOfTrialIssued';
```
Comment (TomElliottFlexion) [inline web-client/src/app.jsx]: Lint issue, says Github

**Review by mrinsin — APPROVED** [review]
Wow thanks for doing this! Have you had a chance to verify visually with one or two PDFs if look the same as before?

File: shared/src/business/utilities/htmlGenerator/documents/_standing-pretrial-order.scss [hunk 3]
```diff
@@ -0,0 +1,48 @@
+#standing-pretrial-order {
+  p {
+    font-size: 14px;
+    line-height: 15px;
+  }
+
+  .standing-pretrial-order-small-case {
```
Comment (TomElliottFlexion) [inline shared/src/business/utilities/htmlGenerator/documents/_standing-pretrial-order.scss]: Does the updated styling also work correctly with the `GettingReadyForTrialChecklist`?

Comment (TomElliottFlexion) [reply shared/src/business/utilities/htmlGenerator/documents/_standing-pretrial-order.scss]: I believe the `#standing-pretrial-order` id will have to be added to `StandingPretrialOrderForSmallCase.jsx`. `GettingReadyForTrialChecklist` also references this class outside a `#standing-pretrial-order`

**Review by TomElliottFlexion — APPROVED** [review]

File: shared/src/business/utilities/htmlGenerator/documents/_standing-pretrial-order.scss (hunk 3 above)
Comment (codyseibert) [reply shared/src/business/utilities/htmlGenerator/documents/_standing-pretrial-order.scss]: I'll refactor that

**Comment by codyseibert** [comment]
> Wow thanks for doing this! Have you had a chance to verify visually with one or two PDFs if look the same as before?

yeah I think I've compared them all

**Pull request merged**